compiler/testData/ir/irText/types/castsInsideCoroutineInference.kt

```diff
@@ -1,4 +1,3 @@
-// IGNORE_BACKEND_FIR: JVM_IR
```

Comment:
This is probably not related to this commit, looks like someone forgot to update the expected data for this test.